templates/highly-efficient-record-processor.cfg.xml.erb

```diff
@@ -29,10 +28,10 @@
             <cadf:attachments>
                 <cadf:attachment name="auditData" contentType="ua:auditData">
                     <cadf:content>
-                        <ua:auditData>
+                        <ua:auditData version="1">
                             <ua:region>{{region}}</ua:region>
                             <ua:dataCenter>{{dataCenter}}</ua:dataCenter>
-                            <ua:methodLabel>TODO</ua:methodLabel>
+                            <ua:methodLabel>{{methodLabel}}</ua:methodLabel>
```

Comment:
the repose herp filter provides the value at runtime.  It's part of the configuration to repose -- nothing to do with puppet.